.github/workflows/require_labels.yml

```diff
@@ -11,6 +11,9 @@ jobs:
       GH_TOKEN: ${{ github.token }}
     steps:
       - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
```

Comment:
The default Ruby on ubuntu-latest must be older, since it was failing on `Array#intersect`.